apps/cow-amm-deployer/tsconfig.json

```diff
@@ -6,6 +6,10 @@
       "#/*": ["./src/*"]
     }
   },
+  "content": [
+    "./node_modules/@bleu-fi/ui/dist/**/*",
```

Comment:
🤔 do we need this here?

Reply:
I don't think so actually, let me remove it.